src/v3/src/transformer/layout/oktaVerify/transformOktaVerifyFPLaunchAuthenticator.ts

```diff
@@ -62,5 +64,24 @@ export const transformOktaVerifyFPLaunchAuthenticator: IdxStepTransformer = ({
     launchAuthenticatorButton,
   ];
 
+  if (!getCookieUserAuthenticated()) {
+    const existingBackLink: LinkElement = uischema.elements.find(
+      (e) => e.type === 'Link' && (e as LinkElement).options.step === 'cancel',
+    ) as LinkElement;
```

Comment:
Is this logic needed? Buttons and links are added later in the flow of the schema transformation process, so at this point, there should never be a link that exists in the elements array. Was this added for a specific reason?

Reply:
this is a safety net, to avoid adding duplicate links

Reply:
I see, based on how the transformation layer is structured, this is the only place where this link can be added, if someone in the future makes changes that incorrectly adds this link (eg. somewhere later in the flow) the tests will fail since it looks for exactly one link. I don't think this is necessary, I think it adds confusion because someone unfamiliar may come here and think links for this flow are/can be added somewhere higher in the flow.